docs/src/examples/components/Checkbox/Types/CheckboxListboxExample.shorthand.tsx

```diff
@@ -0,0 +1,13 @@
+import { List, Checkbox, checkboxMultiselectBehavior } from '@fluentui/react'
+import * as React from 'react'
+
+const CheckboxLisboxExample = () => (
+  <List selectable aria-multiselectable={true}>
```

Comment:
Is `List` component the best choice? I understand that from accessibility point it may be, but from styling point, if we decide to add some styles on the List component int he future, or if any theme has, those will be applied here as well...

Reply:
Yeah, may be we should introduce `CheckboxGroup` for this...

Reply:
I am not sure about this - the pattern is multi select listbox, so it is a combination of List and Checkbox. It would be more clear if we would introduce a "neutral" variabt of List, in case if we needed to add some styles to List